src/cli/client.rs

```diff
@@ -331,6 +332,32 @@ impl KrillClient {
                 Ok(ApiResponse::BgpAnalysisSuggestions(suggestions))
             }
 
+            CaCommand::AspasList(handle) => {
```

Comment:
For ROAs you used the full name e.g. `RouteAuthorizationsUpdate` but here you use the abbreviation ASPA. This is inconsistent. Is this deliberate because names like `AutonomousSystemProviderAuthorizationsUpdate` was felt to be too long or is this inconsistency accidental?

Reply:
Indeed, this is deliberate and writing it out would be too long.

Additionally.. for ROAs I try to talk about "RouteAuthorization" when it comes to configured prefixes which are to be put on one or *more* ROAs, potentially together with other configured prefixes. So there is a many-many mapping going on and using "Roa" there would be more confusing because typically ROA is the *object*.

For ASPA that is slightly different because all providers for a 'customer' ASN MUST (RFC) always appear on a single object. Even if you would get multiple ASPA objects if the customer ASN is held in more than one resource class.. this is much like a one-many mapping.